guides/understanding_masking_and_padding.py

```diff
@@ -264,6 +264,11 @@ def compute_mask(self, inputs, mask=None):
 
 print(mask)
 
+"""
+Note: For more details about format limitations related to masking, see the 
+[serialization guide](/guides/serialization_and_saving).
```

Comment:
I think this link will be broken everywhere except keras.io (github, colab, tensorflow.org, since the paths are all different.

Please use a full URL.

Reply:
`/guides/name/` is used in https://github.com/keras-team/keras-io/blob/master/guides/training_with_built_in_methods.py#L27

The guide has the correct links:
https://www.tensorflow.org/guide/keras/train_and_evaluate#introduction
https://keras.io/guides/training_with_built_in_methods/#introduction